backend/src/main/java/io/debezium/configserver/model/ConnectConnectorConfigResponse.java

```diff
@@ -0,0 +1,40 @@
+package io.debezium.configserver.model;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ConnectConnectorConfigResponse {
+
+    public ConnectConnectorConfigResponse() {
+    }
+
+    private String name;
```

Comment:
Member order is a bit odd, this should be first. Also can i be final?

Reply:
Can't be final because of the default constructor. but I will re-arrange the member.